src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -169,9 +178,9 @@ class boss_gormok : public CreatureScript
 
             void Reset() override
             {
-                events.ScheduleEvent(EVENT_IMPALE, urand(8*IN_MILLISECONDS, 10*IN_MILLISECONDS));
-                events.ScheduleEvent(EVENT_STAGGERING_STOMP, 15*IN_MILLISECONDS);
-                events.ScheduleEvent(EVENT_THROW, urand(15*IN_MILLISECONDS, 30*IN_MILLISECONDS));
+                events.ScheduleEvent(EVENT_IMPALE, randtime(Seconds(8), Seconds(10)));
```

Comment:
ScheduleEvent has a new overload since 2174c5d7dd9669384b2e68b8b406a3173042d7d9.
You can simplify to `events.ScheduleEvent(EVENT_IMPALE, Seconds(8), Seconds(10))`.